packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -117,7 +118,17 @@ void main() {
     expect(_getRawMaterialButton(tester).splashColor, splashColor);
   });
 
+  // The feature checked by this test has been deprecated, see
+  // flutter.dev/go/remove-fab-accent-theme-dependency. This test will be
+  // removed in the future.
   testWidgets('FloatingActionButton foreground color uses iconAccentTheme if no widget or widget theme color is specified', (WidgetTester tester) async {
+    final DebugPrintCallback oldPrint = debugPrint;
+    final List<String> log = <String>[];
+    debugPrint = (String message, { int wrapWidth }) {
+      log.add(message);
+    };
+    debugPrintBuildScope = true;
```

Comment:
What does this do?

Reply:
Good point, it's not needed in this case.